submissions/ogennaOgbogu/htmlForms/signUp.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Sign Up</title>
+        <link rel="stylesheet" type="text/css" href="..//CSS/signupStylesheet.css">
```

Comment:
Are you sure this connect is correct? @Oge-nna

Reply:
Hi @ToluAkin, I don't understand your question, kindly explain.

Reply:
I mean the path to the css file @Oge-nna . There is a double slash.